app/src/main/java/ch/sdp/vibester/activity/TypingGameActivity.kt

```diff
@@ -246,9 +247,28 @@ class TypingGameActivity : AppCompatActivity() {
         else{
             if(runnable !=null ){
                 h.removeCallbacks(runnable!!);}
-            val i = Intent(this, EndBasicGameTemporary::class.java)
-            i.putExtra("score", gameManager.getScore().toString())
-            startActivity(i)
+            switchToEnding(gameManager)
         }
     }
+
+    private fun switchToEnding(gameManager:GameManager) {
```

Comment:
Maybe refactor the creations of the ArrayList<String>and the calls to putStringArrayListExtra into a separate function? (Unless the values we will use instead of the hardcoded ones are of types other than String). The rest of the function looks good to me.